packages/react-instantsearch-dom/src/hooks/useAnswers.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect, useMemo, useContext } from 'react';
```

Comment:
This is the first time that we introduce React Hooks in the codebase and they're compatible from React 16.8.0. Our `react*` peer dependencies start from 16.3.0, which would throw here: https://github.com/algolia/react-instantsearch/blob/cd530682856d8ffd29db202a61fe83f52710ee05/packages/react-instantsearch-dom/package.json#L51-L52

Bumping our `react*` peer dependencies would be a major bump in React InstantSearch.

How do you plan to make it work in the current React InstantSearch version?

Reply:
could we give a warning if this new widget is used in react pre 16.8? Since this is an experimental widget, there might be different tradeoffs

Reply:
https://github.com/algolia/react-instantsearch/pull/2996/commits/2f8a79f618e2c57a3de732d456b95888f05f3861

what do you think?